src/Sulu/Bundle/TranslateBundle/Tests/DataFixtures/Files/sulu.en.xlf

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
-  <file source-language="en-GB" target-language="en" datatype="plaintext" original="file.ext">
+  <file source-language="en" target-language="en" datatype="plaintext" original="file.ext">
```

Comment:
@wachterjohannes strange. i dont know we this is always generated. Nevermind I'll delete it and push again